Gemfile

```diff
@@ -5,7 +5,9 @@ ruby '2.6.1'
 
```

Comment:
- Xóa toàn bộ comment trong file này
- Dùng " thay cho ' trong toàn bộ file này